clusters/app.ci/prow-monitoring/additional-scrape-configs_secret.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: additional-scrape-configs
+  namespace: prow-monitoring
+stringData:
+  prometheus-additional.yaml: |
+    - job_name: blackbox
+      metrics_path: /probe
+      params:
+        module: [http_2xx]
+      static_configs:
+        - targets:
+          - https://registry.build02.ci.openshift.org
```

Comment:
We might as well add in the API, too, right? Anything we self-manage

Reply:
Yes.
This is POC.
I will add all of them later (when we are done with the their certs):

Prow:
- deck
- bokos
...

build01:
- prometheus
- registry

build02:
...